ddl/ddl_worker.go

```diff
@@ -1066,6 +1090,7 @@ func updateSchemaVersion(t *meta.Meta, job *model.Job) (int64, error) {
 				diff.AffectedOpts = buildPlacementAffects(oldIDs, oldIDs)
```

Comment:
cc @djshow832 I'm not sure whether `ActionRecoverSchema` ddl job need this to update placement rule cache, PTAL

Reply:
@xhebox @lcwangchao  PTAL

Reply:
@lcwangchao Do we implement it in this PR or not? I think we could recover placement rules without much problem..